examples/xrt_two_kernels.py

```diff
@@ -0,0 +1,109 @@
+from pathlib import Path
+
+import numpy as np
+
+from xaiepy import pyxrt, xclbinutil
+from xaiepy.pyxrt import ert_cmd_state
+
+XCLBIN_FILE = Path(__file__).parent.absolute() / f"twokernels_new.xclbin"
+xclbinutil.merge_xclbins(
+    "pi.xclbin", "twopi.xclbin", "twopi_kernel.json", str(XCLBIN_FILE)
+)
```

Comment:
@nirvedhmeshram you can ignore basically everything in this PR except for this line. The gist is the PR implements two ways to merge xclbins:

1. by binding the xclbinutil APIs directly (in `xaiepy/xclbinutil.cpp`, which is a pybind extension)
2. by shelling out (in `examples/dump.py`)

Currently it does work in that you get numerical accuracy (the right numbers come out) but there is some kind of race condition or something somewhere because it's slow (wouldn't be suprised if it were slower than actually loading individual xclbins). To see this you can remove the `break` and uncomment the second half of the loop below. [mlir-aie](https://github.com/Xilinx/mlir-aie/pull/1508) has a PR up (that I emulated) where the same thing does not occur but `dump.py` is literally their same code :shrug:. 

FYI I spent like 20 hours on this and it was horrible but what I learned was that the xclbins do literally nothing that isn't immediately [undone by the driver](https://github.com/amd/xdna-driver/blob/2157a343cd25c139e34e63837f974c6d6c7eb1a9/src/shim/hwctx.cpp#L147). That means eventually we can get rid of all of this.

Reply:
I tested just running the same kernel twice from an xclbin that only has one kernel (in `examples/xrt.py`) and I see the same thing (actually that just fails). I have no clue but it would not surprise me if the xrt python bindings are just sloppy with respect to ref counting (and hence direct use in C++ doesn't face the problem). Obviously at this point it's safe to say the problem could be absolutely anywhere in the stack :shrug:.

Reply:
refactored to drop the context every time and now we have this

```
ert_cmd_state.ERT_CMD_STATE_ERROR
3.14

ert_cmd_state.ERT_CMD_STATE_COMPLETED
6.28

ert_cmd_state.ERT_CMD_STATE_COMPLETED
3.14

ert_cmd_state.ERT_CMD_STATE_COMPLETED
6.28

ert_cmd_state.ERT_CMD_STATE_COMPLETED
3.14

ert_cmd_state.ERT_CMD_STATE_COMPLETED
6.28

ert_cmd_state.ERT_CMD_STATE_COMPLETED
3.14

ert_cmd_state.ERT_CMD_STATE_COMPLETED
6.28
```

So something goes wrong the first time but not subsequent times. Note, I made a similar sort of change to the single cdo example and this isn't the behavior (no fails there).